src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -146,7 +147,10 @@ public function editAction(Post $post, Request $request)
 
         if ($form->isSubmitted() && $form->isValid()) {
             $post->setSlug($this->get('slugger')->slugify($post->getTitle()));
-            $entityManager->flush();
+            $entityManager->transactional(function ($entityManager) {
```

Comment:
I appreciate your contribution ... but using `$entityManager->transactional` is "light years" ahead of what we want to teach in the Symfony Demo app. I'm sorry but we can't use this obscure feature. We'll need to find a much simpler alternative solution. Thanks for understanding it.